infrastructure/main.tf

```diff
@@ -65,14 +66,28 @@ module "claim-store-api" {
     // staff notifications
     STAFF_NOTIFICATIONS_SENDER = "noreply@reform.hmcts.net"
     STAFF_NOTIFICATIONS_RECIPIENT = "civilmoneyclaims+cnp@gmail.com"
+
+    // feature toggles
+    CLAIM_STORE_TEST_SUPPORT_ENABLED = "${var.env == "prod" ? "false" : "true"}"
```

Comment:
do you have to use `"` to wrap "prod", "false" and "true"?

Reply:
yeah you do